platform-core-module/src/main/java/com/softicar/platform/core/module/file/stored/attribute/StoredFileInput.java

```diff
@@ -23,7 +23,7 @@ public StoredFileInput() {
 	}
 
 	@Override
-	public AGStoredFile getValueOrThrow() throws EmfInputException {
+	public AGStoredFile getValueOrThrow() throws DomInputException {
```

Comment:
I renamed `EmfInputException` to `DomInputException` and moved it from EMF to DOM